homeassistant/components/isy994/helpers.py

```diff
@@ -107,6 +108,11 @@ def _check_for_insteon_type(
                 hass_isy_data[ISY994_NODES][LIGHT].append(node)
                 return True
 
+            # Thermostats, which has a "Heat" and "Cool" sub-node on address 2 and 3
+            if platform == CLIMATE and str(node.address[-1]) in ["2", "3"]:
```

Comment:
Missed these on the first pass, but it would be nice to clean up the magic numbers